stumpy/aamp.py

```diff
@@ -194,13 +234,30 @@ def _aamp(
         Set to `True` if this is a self-join. Otherwise, for AB-join, set this to
         `False`. Default is `True`.
 
+    k : int
+        The number of top `k` smallest distances used to construct the matrix profile.
+        Note that this will increase the total computational time and memory usage
+        when k > 1.
+
     Returns
     -------
-    P : numpy.ndarray
-        Matrix profile
+    profile : numpy.ndarray
```

Comment:
@seanlaw 
Yeah...I missed it! Btw, I think we should use the names `out1, out2, ...`. This is because the final output is calculated inside the return statement.

Reply:
I'm good with that